src/components/pages/devices/flyouts/SIMManagement/SIMManagement.js

```diff
@@ -74,6 +82,12 @@ export class SIMManagement extends LinkedComponent {
               </Section.Content>
             </Section.Container>
           }
+          <BtnToolbar>
+            {!isPending && <Btn primary={true} disabled={!provider} onClick={this.showProvider} type="submit">{t('devices.flyouts.new.apply')}</Btn>}
+            {!isPending && <Btn svg={svgs.cancelX} onClick={onClose}>{t('devices.flyouts.new.cancel')}</Btn>}
```

Comment:
If both of these buttons have the same functionality might as well put the condition inside to just change the text.